### DISCUSSION TITLE
Person Record Updates

### DISCUSSION THREAD

**Discussion by michael-wynne-wsu** [MukurtuCMS/Mukurtu-CMS#22] [Feature Enhancement]
Requested changes, additions, updates to Person Record feature.

**Comment by MonicaGalassi** [comment]
@michael-wynne-wsu I was wondering if there can be a couple of out-of-the-box templates for the person record to visualise content? For example one with a photo on the left and a banner, or photo at the bottom of the page, etc. I think t's an amazing feature but last time we tried to use it we couldn't really align the content properly.

**Comment by MonicaGalassi** [comment]
Tree visualisation, maybe with the option of being linked to a map would be awesome as well. I think this could be helpful to visualise communities and families that have been moved from one area to another (for example in missions and reserves)
